usr/lib/cca_stdll/cca_specific.c

```diff
@@ -3310,6 +3323,117 @@ CK_RV token_specific_set_attrs_for_new_object(STDLL_TokData_t *tokdata,
 }
 #endif /* NO_PKEY */
 
+typedef struct {
+    CK_BBOOL card_level_set;
+    struct {
+        unsigned int ver;
+        unsigned int rel;
+        unsigned int mod;
+    } min_card_version;
+} cca_min_card_version_t;
+
+#define CCA_STATCCA_CCA_VERSION_OFFSET         24
+#define CCA_STATCCA_CCA_VERSION_LENGTH          8
+#define MAX_INT                                0xFFFFFFFF
```

Comment:
Move these defines to cca_stdll.h near the other offset defines

And use UINT_MAX from limits.h

Reply:
Done.